procurement/views.py

```diff
@@ -126,6 +126,11 @@ def get_context_data(self, **kwargs):
 
         if self.request.GET.get("region"):
             context["region_choice"] = self.request.GET.get("region")
+        
+        if self.request.GET.get("council_exact"):
+            council = self.request.GET.get("council_exact")
+            if (council,) not in Council.objects.all().values_list("name"):
```

Comment:
`if not Council.objects.filter(name__iexact=council.lower()).exists()` is a slightly more efficient way to do this.

Reply:
Thank you! This has been changed.